doc/api/dns.md

```diff
@@ -195,6 +196,15 @@ Uses the DNS protocol to resolve mail exchange records (`MX` records) for the
 contain an array of objects containing both a `priority` and `exchange`
 property (e.g. `[{priority: 10, exchange: 'mx.example.com'}, ...]`).
 
+## dns.resolveNaptr(hostname, callback)
+
+Uses the DNS protocol to resolve regular expression based records (`NAPTR`
+records) for the `hostname`. The `addresses` argument passed to the `callback`
```

Comment:
I don't think it's redundant, because they don't really overlap. The callback signature says what arguments the callback will provide. Currently, we just know that there is an `addresses` argument that has a certain schema. What about an error parameter, etc?